src/spec-node/featuresCLI/publish.ts

```diff
@@ -82,10 +84,16 @@ async function featuresPublish({
             process.exit(1);
         }
 
-        if (! await doPublishCommand(f.version, featureRef, outputDir, output, collectionType)) {
+        const publishResult = await doPublishCommand(f.version, featureRef, outputDir, output, collectionType);
```

Comment:
As `templates publish command` also uses the same `doPublishCommand()`. Wondering, if you'd also like to output the result for Templates as well?

You'd likely need to make similar changes like this file to [src/spec-node/templatesCLI/publish.ts](https://github.com/devcontainers/cli/blob/main/src/spec-node/templatesCLI/publish.ts)

Reply:
Good point, might as well make those two consistent 👍

Reply:
<img width="1456" alt="image" src="https://user-images.githubusercontent.com/23246594/205122980-b5640ea6-c282-461b-8b87-53273e04014a.png">

Reply:
🎉